### ISSUE TITLE
Add feature to allow testing regular functions, classes

### ISSUE THREAD

**Issue by ptutak** [inmanta/pytest-inmanta#37]
Right now there is no possibility to test regular functions and classes from the level of a project.

It would be very useful to add a feature that would enable testing any item in the plugins/__init__.py

This should be done by either
1. a normal import
2. a method on the project fixture which return the module 

e.g.

```python
def my_function(arg):
    pass

class MyClass:
    pass
```
```python
mymodule = project.do_import('mymodule')

mymodule.my_function
```

**Comment by wouterdb** [comment]
We could probably even do without the annotation

**Comment by wouterdb** [comment]
This will be resolved by inmanta#507
